ansible/playbooks/roles/compatibility_layer/tasks/install_packages.yml

```diff
@@ -1,15 +1,29 @@
 # Install a specified list of sets and packages.
 ---
-- name: Install package set
+- name: Install package set {{ package_sets }}
   portage:
     package: "@{{ item }}"
     state: present
   with_items: "{{ package_sets }}"
   tags:
     - set
+  become: yes
 
 - name: Install additional packages
   portage:
     package: "{{ item }}"
     state: present
   with_items: "{{ prefix_packages }}"
+  become: yes
+
+- name: "Get the username running the deployment (not root)"
+  become: no
+  shell: set -o pipefail && who | grep -v root | awk '{print $1}' | uniq
+  register: username_on_host
+  changed_when: True
+
+- name: "Fix permissions after installing as portage/root"
+  become: yes
+  shell: chown -R {{ username_on_host }}:{{ username_on_host }} {{ gentoo_prefix_path }}
```

Comment:
Can this be done with the `file` module and `recurse: yes`?

Reply:
Yup. I should fix that.

Reply:
Trying it with this now:
```
- name: "Fix permissions after installing as portage/root"
  become: yes
  file:
    owner: "{{ username_on_host }}"
    group: "{{ username_on_host }}"
    path: "{{ gentoo_prefix_path }}"
    recurse: true

```